src/Compilers/Core/Portable/Operations/OperationWalker.cs

```diff
@@ -40,454 +37,20 @@ public override void Visit(IOperation operation)
             }
         }
 
-        internal override void VisitNoneOperation(IOperation operation)
+        public override void DefaultVisit(IOperation operation)
         {
             if (operation is IOperationWithChildren operationWithChildren)
             {
                 VisitArray(operationWithChildren.Children);
             }
         }
 
-        public override void VisitBlockStatement(IBlockStatement operation)
```

Comment:
Operation and walker will be changed at the same time, so there should be no visible behavior difference for users.